h2o-core/src/test/java/water/rapids/ast/prims/misc/SetClusterPropertyTaskTest.java

```diff
@@ -0,0 +1,36 @@
+package water.rapids.ast.prims.misc;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+import water.MRTask;
+import water.TestUtil;
+import water.rapids.Rapids;
+import water.rapids.Val;
+import water.rapids.vals.ValStr;
+
+import static org.junit.Assert.*;
+
+public class SetClusterPropertyTaskTest extends TestUtil {
+
+  @BeforeClass
+  public static void setup() {
+    stall_till_cloudsize(5);
+  }
+
+  @Test
+  public void testSetClusterProperty() {
```

Comment:
As this is a Junit test, the property being properly set in the environment can be tested as well by System.getEnv()

Reply:
It is tested on line 27. Do you have something else in mind?

Reply:
No, I originally added a second check after the second rapids call, but that is indeed in vain.